src/FM.LiveSwitch.Mux.Standard/FileUtility.cs

```diff
@@ -60,7 +60,7 @@ private static bool IsLocked(IOException ex)
 
         public static bool Exists(string path)
         {
-            return path != null && File.Exists(path) && new FileInfo(path).Length > 0;
+            return path != null && File.Exists(path);
```

Comment:
Intentional?

Reply:
Yes, I think it was intentional - this is result of a merge from master

Reply:
@olegkabanov I think we want to revert this change then and keep what is in `master`?